capstone/src/main/java/com/google/univiz/scorecard/CollegeData.java

```diff
@@ -0,0 +1,23 @@
+package com.google.univiz.scorecard;
+
+/**
```

Comment:
does this need any JavaDoc? not sure

Reply:
I would leave it in. it's good to know what this interface intends to do...